### DISCUSSION TITLE
Configuring git hub pages on a custom domain.

### DISCUSSION THREAD

**Discussion by iantierney** [community/community#28183] [Apps, API and Webhooks]
Hey,
I bought a domain from godaddy and tried to point my git hub page using this
[guide:](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain)
It didn't work.
I spoke to the godaddy people who do this all day and they said the above git hub pages instructions make no sense.

Has anyone managed to do this?

Ian

**Comment by jfullstackdev** [comment]
this is what I'm seeing on my end 

![image](https://user-images.githubusercontent.com/47092464/182620466-986d8e49-24bf-4c0f-abdf-8a03c39b2715.png)

what is the error message in your setup ? or maybe it will take 24 hours

**Comment by iantierney** [comment]
Yeah same.
I spoke to godaddy and they said you need the "name servers". No mention of that in the GitHub guide.
